plugin/pathvirt/README

```diff
@@ -0,0 +1,95 @@
+# pathvirt (Path Virtualization DMTCP Plugin)
+
+pathvirt is a DMTCP plugin that virtualizes access to file system paths for
+programs run under DMTCP. This allows path access to be transparently
+redirected elsewhere on the file system on DMTCP restart.
+
+## Usage
+
+pathvirt operates on the basis of path prefixes and is interfaced with using
+the DMTCP_PATH_PREFIX environment variable.
```

Comment:
Did we agree that we will only support `DMTCP_ORIGINAL_PATH_PREFIX` and `DMTCP_NEW_PATH_PREFIX`?  If so, let's modify the sentence above.